tests/DCPS/ReadCondition/ReadConditionTest.cpp

```diff
@@ -106,6 +106,7 @@ int run_test_instance (int argc, ACE_TCHAR *argv[])
                  << ret << endl;
             passed = false;
             done = true;
+            break;
```

Comment:
Yes, but this one is one the I observed failing.  I think we'll need further changes, but I wanted to get this much in first.